packages/shared/routes/setup/Legal.svelte

```diff
@@ -53,26 +48,695 @@
+                        We may process data about your use of the App in order to enable the operation and use of the App, as well
+                        as for the administration and management of the App and the services offered via it.
+                    </Text>
+
+                    <Text type="h2" classes="mt-8 mb-5">Lawful Basis for Processing of Data</Text>
+                    <Text type="p" secondary classes="mb-4">
+                        In processing your Personal Data in connection with the purposes set out in this Privacy Policy, we may
+                        rely on one or more of the following legal bases, depending on the circumstances:
+                    </Text>
+                    <Text type="p" secondary classes="mb-4">
+                        <ul>
+                            <li>
+                                <Text type="p" secondary classes="mb-4">
+                                    we have obtained your explicit prior consent to the processing (this legal basis is only used
+                                    in relation to processing that is entirely voluntary &ndash; it is not used for processing
+                                    that is necessary or obligatory in any way);
+                                </Text>
+                            </li>
+                            <li>
+                                <Text type="p" secondary classes="mb-4">
+                                    the processing is necessary in connection with any contractual relationship that you may enter
+                                    into with us;
+                                </Text>
+                            </li>
+                            <li>
+                                <Text type="p" secondary classes="mb-4">the processing is required by the applicable law;</Text>
+                            </li>
+                            <li>
+                                <Text type="p" secondary classes="mb-4">
+                                    the processing is necessary to protect the vital interests of any individual; or
+                                </Text>
+                            </li>
+                            <li>
+                                <Text type="p" secondary classes="mb-4">
+                                    we have a legitimate interest in carrying out the processing for the purpose of managing,
+                                    operating or promoting our business, and that legitimate interest is not overridden by your
+                                    interests, fundamental rights, or freedoms.
+                                </Text>
+                            </li>
+                        </ul>
+                    </Text>
+
+                    <Text type="h2" classes="mt-8 mb-5">Use of Cookies</Text>
+                    <Text type="p" secondary classes="mb-4">We do not use cookies in the App.</Text>
```

Comment:
But we do use localStorage which is included in the same `cookie` laws

https://law.stackexchange.com/questions/30739/do-the-gdpr-and-cookie-law-regulations-apply-to-localstorage